pkg/api/batch/operation.go

```diff
@@ -28,14 +28,14 @@ type Operation struct {
 	// signed data for the operation
 	SignedData *model.JWS `json:"signedData"`
 
-	// operation patch data
-	PatchData *model.PatchDataModel `json:"patchData"`
+	// operation delta
+	Delta *model.DeltaModel `json:"delta"`
 
 	// suffix data
 	SuffixData *model.SuffixDataModel `json:"suffixData"`
 
-	// encoded patch data
-	EncodedPatchData string `json:"encodedPatchData"`
+	// encoded delta
+	EncodedDelta string `json:"encodedDelta"`
```

Comment:
they use underscores rather than snake case.

Reply:
Operation is our internal model